### PR TITLE
Removed image values for MongoDB and PostgresDB charts in integration tests

### PR DESCRIPTION
**Pull request by shahpratikr**
## Change Overview

- removed usage of kanister images while installing MongoDB and PostgresDB helm charts
- fixed minor issues in formatting

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [x] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

Ran integration tests for Mongodb and Postgresql using following commands
`./build/integration-test.sh ^PostgreSQL$` and `./build/integration-test.sh ^MongoDB$`

### CODE REVIEW

**Comment by PrasadG193** [comment]
@shahpratikr could you please add the test plan?

**Comment by shahpratikr** [comment]
> @shahpratikr could you please add the test plan?

added

**Review by akankshakumari393 — APPROVED** [review]
LGTM

File: pkg/app/mongodb.go [hunk 1]
```diff
@@ -63,9 +63,6 @@ func NewMongoDB(name string) HelmApp {
 			Chart:    "mongodb",
 			Values: map[string]string{
 				"architecture":     "replicaset",
-				"image.registry":   "ghcr.io",
-				"image.repository": "kanisterio/mongodb",
-				"image.tag":        "v9.99.9-dev",
```
Comment (viveksinghggits) [inline pkg/app/mongodb.go:68]: @shahpratikr should we remove these container images from our goreleaser workflow as well? If we are not consuming them anymore.

**Review by viveksinghggits — COMMENTED** [review]
a comment, otherwise looks good.

File: pkg/app/mongodb.go (hunk 1 above)
Comment (shahpratikr) [reply pkg/app/mongodb.go:68]: @viveksinghggits this image is used in blueprints right?
```
./examples/mongodb/blueprint-v2/mongo-blueprint.yaml:        image: ghcr.io/kanisterio/mongodb:0.85.0
./examples/mongodb/blueprint-v2/mongo-blueprint.yaml:        image: ghcr.io/kanisterio/mongodb:0.85.0
./examples/mongodb/blueprint-v2/mongo-blueprint.yaml:        image: ghcr.io/kanisterio/mongodb:0.85.0
```

Comment (viveksinghggits) [reply pkg/app/mongodb.go:68]: If we don't use them in application, do we need them in any of the blueprints? Are we just using them in the v2 blueprints?

Comment (shahpratikr) [reply pkg/app/mongodb.go:68]: We are using this image at multiple places. This was just the snippet from grep command. If you run `grep "kanisterio/mongodb" -rw .` inside kanister repo, you will get all the files where we are using these images

**Review by pavannd1 — COMMENTED** [review]
In the future, we should create two different PRs when changes are done to different workflows/examples, etc. It makes reviewing easier and reverts too in case of issues in one workflow.

File: pkg/app/postgresql-deploymentconfig.go [hunk 2]
```diff
@@ -173,7 +173,7 @@ func (pgres *PostgreSQLDepConfig) Count(ctx context.Context) (int, error) {
 
 	out := strings.Fields(stdout)
 	if len(out) < 4 {
-		return 0, fmt.Errorf("Unknown response for count query")
+		return 0, fmt.Errorf("unknown response for count query")
```
[the remark this answers is not in the document]
Comment (shahpratikr) [reply pkg/app/postgresql-deploymentconfig.go:176]: was seeing `error strings should not be capitalized (ST1005)` this warning. hence made these change

File: pkg/app/rds_postgres.go [hunk 3]
```diff
@@ -95,7 +95,7 @@ func (pdb *RDSPostgresDB) Init(ctx context.Context) error {
 	if pdb.region == "" {
 		pdb.region, ok = os.LookupEnv(aws.Region)
 		if !ok {
-			return fmt.Errorf("Env var %s is not set", aws.Region)
+			return fmt.Errorf("env var %s is not set", aws.Region)
```
[the remark this answers is not in the document]
Comment (shahpratikr) [reply pkg/app/rds_postgres.go:98]: was seeing `error strings should not be capitalized (ST1005)` this warning. hence made these change

File: pkg/app/rds_postgres.go [hunk 4]
```diff
@@ -107,11 +107,11 @@ func (pdb *RDSPostgresDB) Init(ctx context.Context) error {
 
 	pdb.accessID, ok = os.LookupEnv(aws.AccessKeyID)
 	if !ok {
-		return fmt.Errorf("Env var %s is not set", aws.AccessKeyID)
+		return fmt.Errorf("env var %s is not set", aws.AccessKeyID)
```
[the remark this answers is not in the document]
Comment (shahpratikr) [reply pkg/app/rds_postgres.go:110]: was seeing `error strings should not be capitalized (ST1005)` this warning. hence made these change

File: pkg/app/mongodb.go (hunk 1 above)
Comment (viveksinghggits) [reply pkg/app/mongodb.go:68]: Like discussed let's raise another PR to with changes to make sure we remove the images from goreleaser that are not used anymore, and app-tools image can be used instead.

**Review by viveksinghggits — APPROVED** [review]

**Pull request merged**